### PR TITLE
Add base library chart

### PR DESCRIPTION
**Pull request by skolobov**

### CODE REVIEW

**Comment by voron** [comment]
I still see `app` instead of `base` in `README.md`, as well as direct chat deployment via helm. Could you pls update readme to get faster  how it's supposed to work?

**Review by dennybaa — COMMENTED** [review]
> I still see `app` instead of `base` in `README.md`, as well as direct chat deployment via helm. Could you pls update readme to get faster how it's supposed to work?

The readme is not crucial.  Moreover it was generated using some tool. And the lease but not last we don't have any solution for generating readmes, whereas in fact I suggest to completely avoid editing readmes by hand...

File: dysnix/base/templates/_validations.tpl [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+{{- end -}}
+
+{{/* Validate value name is given */}}
+{{- define "base.validateValues.nameGiven" -}}
+{{- if not .Values.name -}}
+app: no-name
+    You did not specify the application name for the generic app chart. Please
+    set name.
+{{- end -}}
+{{- end -}}
```
[the remark this answers is not in the document]
Comment (skolobov) [reply dysnix/base/templates/_validations.tpl:26]: Removed

File: dysnix/base/templates/_validations.tpl [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    set image.repository.
+{{- end -}}
+{{- end -}}
+
+{{/* Validate that the container has command or args given */}}
+{{- define "base.validateValues.containerCommandOrArgsGiven" -}}
+{{- if not (or .Values.command .Values.args) -}}
+app: no-command-or-args
+    You did not specify command or args for the application default
+    container. Please set command or args.
```
[the remark this answers is not in the document]
Comment (skolobov) [reply dysnix/base/templates/_validations.tpl:42]: Removed

File: dysnix/base/templates/_names.tpl [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "base.names.name" -}}
+{{- default (.Values.name | default "") .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
```
[the remark this answers is not in the document]
Comment (skolobov) [reply dysnix/base/templates/_names.tpl]: Correct - removed

**Comment by dennybaa** [comment]
resolved since #139

**Pull request closed without merging**